lib/octokit/client/repositories.rb

```diff
@@ -696,6 +715,30 @@ def update_subscription(repo, options = {})
       def delete_subscription(repo, options = {})
         boolean_from_response :delete, "#{Repository.path repo}/subscription", options
       end
+
+      # Check if a repository is a template repository
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @return [Boolean] True if repository is a template repository, false otherwise.
+      def template_repository?(repo)
+        options = ensure_api_media_type(:template_repositories, {})
+        response = get Repository.path(repo), options
+        response.is_template?
+      end
+      alias :template_repo? :template_repository?
+
+      # Change whether a repository is a template repository
+      #
+      # @param repo [String, Hash, Repository] A GitHub repository
+      # @param is_template [Boolean] True if making repo a template, false if making it not a template.
+      # @return [Sawyer::Resource] Repository information
+      def template_repository(repo, is_template)
+        repo = Repository.new(repo)
+        options = { is_template: is_template, name: repo.name }
+        options = ensure_api_media_type(:template_repositories, options)
+        patch "repos/#{repo}", options
+      end
+      alias :template_repo :template_repository
```

Comment:
Following up on your comment https://github.com/octokit/octokit.rb/pull/1133/files#r303087310

I would rather us update the `edit_repository` method than create a new helper method for this. It fits the pattern of the client better.